lib/pengine/utils.c

```diff
@@ -1009,24 +1051,11 @@ unpack_operation(pe_action_t * action, xmlNode * xml_obj, pe_resource_t * contai
 
     g_hash_table_remove(action->meta, "id");
 
-    // Normalize interval to milliseconds
```

Comment:
Let's keep this comment, it's still true -- we're replacing the user-configured interval (in a freeform text format like "10s" or "P10S") with the interval as a number of milliseconds (or removing the interval if it evaluates to 0)

Reply:
Done.